grid-client/graphql/graphql.go

```diff
@@ -37,7 +82,13 @@ func (g *GraphQl) GetItemTotalCount(itemName string, options string) (float64, e
 	cl := &http.Client{
 		Timeout: 10 * time.Second,
 	}
-	countResponse, err := cl.Post(g.url, "application/json", bodyReader)
+
+	baseURL, err := g.baseURL()
```

Comment:
I think we should set the url to use in that client during the initialization only once, and we can call it again in case of error as a recovery step.